src/Requests/Companies/PostRequest.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace TruckersMP\Requests\Companies;
+
+use TruckersMP\Models\CompanyPost;
+use TruckersMP\Requests\Request;
+
+class PostRequest extends Request
+{
+    /**
+     * The ID of the requested company.
+     *
+     * @var int
+     */
+    protected $companyId;
+
+    /**
+     * The DID of the requested post.
```

Comment:
A typo in the comment.

```suggestion
     * The ID of the requested post.
```